shared/constants/app.ts

```diff
@@ -53,6 +53,7 @@ export const MESSAGE_TYPE = {
   PERSONAL_SIGN: 'personal_sign',
   SEND_METADATA: 'metamask_sendDomainMetadata',
   SWITCH_ETHEREUM_CHAIN: 'wallet_switchEthereumChain',
+  TRANSACTION: 'transaction',
```

Comment:
I'm not a huge fan of this, I'll create an alternate ticket to create a new `APPROVAL_TYPE` constant to be explicit.